File: salario/salario.java
import java.util.Scanner;

public class Salario{

	public static void main(String args[]){
    Scanner x = new Scanner(System.in);
		System.out.println("Ingrese su nombre");
		String nombre = x.nextLine();
		System.out.println("Ingresa tu salario por hora");
		double salario = x.nextDouble();
		System.out.println("Cuantas horas trabajas");
		double horas = x.nextDouble();
		double salariofinal = salario*horas;
		if (salariofinal>450000) {
			System.out.println(nombre+" tu salario es: "+salariofinal);
		} else {
			System.out.println(nombre);
		}
	}
}

File: salario/salario.py
Nombre = input("Ingresa tu nombre: ")
SalarioH= int(input("Ingresa el valor por hora de tu salario: "))
Hora= int(input("Cuantas horas trabajaste: "))

SalarioT= SalarioH*Hora

if SalarioT>450000:
  print(Nombre, " tu salario total es de $", SalarioT)
else:
  print(Nombre)
  